baseline/client/js/jspsych-spatial-orientation.js

```diff
@@ -169,11 +169,19 @@ jsPsych.plugins["spatial-orientation"] = (() => {
         }
         // compute timeLimit (duration) from trial.endTime (instant)
         const timeLimit = trial.endTime === null ? null : trial.endTime - Date.now();
+        // build trial params data
+        const paramsData = {
```

Comment:
Now that this is here it would probably be cleaner if we set targetRadians here as well, just so that all of the trial info is set in one place.

Reply:
Good point!